addon/components/stripe-checkout.js

```diff
@@ -67,7 +67,7 @@ export default Component.extend({
     this.get('stripe').close(this);
   },
 
-  init() {
+  willInsertElement() {
```

Comment:
Did you mean `didInsertElement`?

https://guides.emberjs.com/v3.0.0/components/the-component-lifecycle/